dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -569,73 +578,71 @@ DataInterfaceIIDM::importTwoWindingsTransformer(powsybl::iidm::TwoWindingsTransf
   return twoWTfo;
 }
 
-shared_ptr<ThreeWTransformerInterface>
-DataInterfaceIIDM::importThreeWindingsTransformer(powsybl::iidm::ThreeWindingsTransformer & threeWTfoIIDM) {
-  shared_ptr<ThreeWTransformerInterfaceIIDM> threeWTfo(new ThreeWTransformerInterfaceIIDM(threeWTfoIIDM));
-
-  threeWTfo->setBusInterface1(findBusInterface(threeWTfoIIDM.getLeg1().getTerminal().get()));
-  threeWTfo->setVoltageLevelInterface1(findVoltageLevelInterface(threeWTfoIIDM.getLeg1().getTerminal().get().getVoltageLevel().getId()));
-
-  threeWTfo->setBusInterface2(findBusInterface(threeWTfoIIDM.getLeg2().getTerminal().get()));
-  threeWTfo->setVoltageLevelInterface2(findVoltageLevelInterface(threeWTfoIIDM.getLeg2().getTerminal().get().getVoltageLevel().getId()));
-
-  threeWTfo->setBusInterface3(findBusInterface(threeWTfoIIDM.getLeg3().getTerminal().get()));
-  threeWTfo->setVoltageLevelInterface3(findVoltageLevelInterface(threeWTfoIIDM.getLeg3().getTerminal().get().getVoltageLevel().getId()));
-
-  if (threeWTfoIIDM.getLeg1().getCurrentLimits()) {
-    powsybl::iidm::CurrentLimits& currentLimits = threeWTfoIIDM.getLeg1().getCurrentLimits();
-
-    // permanent limit
-    if (!std::isnan(currentLimits.getPermanentLimit())) {
-      shared_ptr<CurrentLimitInterfaceIIDM> cLimit(new CurrentLimitInterfaceIIDM(currentLimits.getPermanentLimit(), std::numeric_limits<unsigned long>::max()));
-      threeWTfo->addCurrentLimitInterface1(cLimit);
+void
+DataInterfaceIIDM::convertThreeWindingsTransformers(powsybl::iidm::ThreeWindingsTransformer & ThreeWindingTransformer) {
+  const string VLId = ThreeWindingTransformer.getId() + "_VL";
+  const string busId = ThreeWindingTransformer.getId() + "_BUS";
+  string countryStr;
+  if (ThreeWindingTransformer.getSubstation().getCountry())
+    countryStr = powsybl::iidm::getCountryName(ThreeWindingTransformer.getSubstation().getCountry().get());
+  shared_ptr<VoltageLevelInterface> vl(new FictVoltageLevelInterfaceIIDM(VLId, ThreeWindingTransformer.getRatedU0(), countryStr));
+  network_->addVoltageLevel(vl);
+  voltageLevels_[vl->getID()] = vl;
+  shared_ptr<BusInterface> fictBus(new FictBusInterfaceIIDM(busId, ThreeWindingTransformer.getRatedU0(), countryStr));
```

Comment:
Can we add a blank line before this block and after so that the steps are more visible? (add Vl, add Bus, handle legs...)

Reply:
yes!